src/components/molecules/PlayaTime/PlayaTime.tsx

```diff
@@ -20,7 +25,7 @@ export const PlayaTime: React.FC = () => {
   useInterval(() => {
     const pt = dayjs().tz(CURRENT_TIMEZONE);
     setCurrentTime(pt.format("h:mm A"));
-  }, 1000);
+  }, ONE_MINUTE_MS);
```

Comment:
Naming it `ONE_MINUTE_MS` in `settings.ts` it means it should never be adjusted. Using it here instead a new constant named `PLAYA_TIME_REFRESH_MS` is a bit off.

Something more off would be updating the time once per minute though. As I think about it, maybe off by few or even 10 seconds from system time will not be as strange, but with such interval it may be off by at most 59 seconds (in cases `setInterval` itself doesn't lag), thus a possible case:

- Playa time showing 11:36, system time showing 11:37
- Playa time changing to 11:37, a second later system time changing to 11:38

All in all, not a deal breaker, still a rule of thumb:
```
const PLAYA_TIME_REFRESH_MS = ONE_MINUTE_MS / 2;  // or 4 or 6 depending on desired lag/granularity
```

Reply:
replaced by separate constant for playa time - 4af31d2